packages/flutter/lib/src/material/button.dart

```diff
@@ -155,6 +157,16 @@ class MaterialButton extends StatefulWidget {
   /// The color of the button, as printed on the [Material].
   final Color color;
 
+  /// The highlight color of the button.
```

Comment:
"highlight color of the button" doesn't really satisfy our style guide, which says you shouldn't be able to write the doc using nothing but the fully qualified name of the member (in this case "MaterialButton.highlightColor"). I suggest adding some more colour such as explaining what a highlight is, and when it appears, and talking about whether it should be opaque.